site/mkdocs.yml

```diff
@@ -12,6 +12,7 @@ nav:
 - Get Started:
   - get-started/quickstart.md
   - samples/notebook.md
+  - get-started/multicluster.md
```

Comment:
I don't think this is the right place. Suggestions for the sections would be appropriated.

I originally planned for quickstart to have just default install and notebook sample, and add another section for Install for custom installation instructions.

Reply:
How about having it in the tasks section?